File: php/src/common/VegetableSoup.php
<?

require_once 'Soup.php';


class VegetableSoup extends Soup
{


	function __construct()
	{
      $this->soupName = "Vegetable Soup";
      $this->soupIngredients = [
        "1 cup bullion",
        "1/4 cup carrots",
        "1/4 cup potatoes",
      ];
	}
}

File: python/src/common/VegetableSoup.py

define ['Creational/Abstract_Factory/Soup',
],(
Soup
) ->

  class VegetableSoup extends Soup
    constructor : ->
      @soupName = 'Vegetable Soup'
      @soupIngredients = [
        '1 cup bullion',
        '1/4 cup carrots',
        '1/4 cup potatoes',
      ]

  return VegetableSoup
